### ISSUE TITLE
Need to cover the UiaDisconnectProvider call with test scenario

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3393]
* .NET Core Version: 

5.0 after merging https://github.com/dotnet/winforms/pull/3308

* Have you experienced this same bug with .NET Framework?: 

No

**Problem description:**

UiaDisconnectProvider call was added to Control.ReleaseUiaProvider method. This introduces releasing accessibility objects but this functionality is not covered be test. Need to add this.

**Expected behavior:**
UiaDisconnectProvider call should be covered by test.

**Minimal repro:**
Review the Control.ReleaseUiaProvider and UiaDisconnectProvider call within it and observe that this is not covered by tests.

Sample standalone test:
[ConsoleUiaTester.zip](https://github.com/dotnet/winforms/files/4730647/ConsoleUiaTester.zip)

**Comment by vladimir-krestov** [comment]
While working on memory leaks issues we didn't find a way of testing this. So @merriemcgaw, can we close it?

**Comment by merriemcgaw** [comment]
Yep
